src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -37,7 +39,10 @@ const ChatBox: React.FC<ChatboxProps> = ({
   const venueId = useVenueId();
   const [isMessageToTheBarSent, setIsMessageToTheBarSent] = useState(false);
 
-  const usersById = useUsersById();
+  const venueUsersById = useUsersById();
+  const chatUsersById = useSelector(chatUsersSelector) ?? {};
+
+  const usersById = isVenueChat ? venueUsersById : chatUsersById;
```

Comment:
This is slightly different to how it used to be, as we used to only run the selector we wanted, whereas this will run both, but then only use one of them. I'm not really sure of a better way to implement this now that we have 1 hook and 1 selector here.. so it's probably fine to leave it as it is. I just wanted to explicitly note it here.